### PR TITLE
[Docs]Add some descriptions for local setup

### PR DESCRIPTION
**Pull request by luanxuechao**
Add some descriptions for local setup

### CODE REVIEW

**Comment by luanxuechao** [comment]
Hi @josephschorr, this is an issue that I set on the local. I hope I can help other people

**Comment by luanxuechao** [comment]
I have read the CLA Document and I hereby sign the CLA

File: examples/v1/example.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { v1 } from '@authzed/authzed-node';
-
+// set up it on localhost Like this:
```
Comment (vroldanbet) [inline examples/v1/example.js:2]:
```suggestion
// set up it on localhost like this:
```

File: README.md [hunk 2]
```diff
@@ -58,7 +58,8 @@ You will have to provide a your own API Token from the [Authzed dashboard] in pl
 
 ```js
 import { v1 } from '@authzed/authzed-node';
-
+// if your endpoint is localhost
+// const client = v1.NewClient('t_your_token_here_1234567deadbeef','localhost:50051',ClientSecurity.INSECURE_LOCALHOST_ALLOWED);
```
Comment (vroldanbet) [inline README.md:62]:
```suggestion
// const client = v1.NewClient('t_your_token_here_1234567deadbeef', 'localhost:50051', ClientSecurity.INSECURE_LOCALHOST_ALLOWED);
```

File: examples/v1/example.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
+// const client = v1.NewClient('mytokenhere',"localhost:50051",1);
```
Comment (vroldanbet) [inline examples/v1/example.js:3]:
```suggestion
// const client = v1.NewClient('mytokenhere', "localhost:50051",1);
```

**Review by vroldanbet — COMMENTED** [review]
thanks for improving the library! left some nits

**Comment by luanxuechao** [comment]
Hi @vroldanbet, thank you, I re-submit the commit, please review it again.

**Review by vroldanbet — APPROVED** [review]

**Pull request merged**